dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.overrides

```diff
@@ -0,0 +1,14 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+cxfVersion=3.2.6
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2
+
+Bundle-Activator: com.ibm.ws.jaxws23.cxf.rt.bindings.soap.NoOpActivator
```

Comment:
This is probably nit-picking, but I was planning on matching the projects, bundle, and packages names what the jaxrs projects does between versions of the jaxrs feature I.E. com.ibm.ws.jaxws.2.3.common etc etc. Would you mind changing the packages to match that? So com.ibm.ws.jaxws.2.3.cxf.rt.binding.soap.NoOpActivator

Reply:
eclipse will not allow a number by itself in a package name